include/nbl/asset/IMeshPacker.h

```diff
@@ -305,29 +258,119 @@ class IMeshPacker : public MeshPackerBase
         core::vector<Triangle> triangles;
     };
 
-    virtual core::vector<TriangleBatch> constructTriangleBatches(MeshBufferType& meshBuffer) = 0;
+    core::vector<TriangleBatch> constructTriangleBatches(MeshBufferType& meshBuffer)
```

Comment:
MeshBufferType*

pointer not reference

Reply:
why?

Reply:
because it would be extremely weird, we dont use references on refcounted types